addons/point_of_sale/static/src/app/screens/payment_screen/payment_screen.xml

```diff
@@ -64,59 +64,104 @@
-                <span class="partner-name">
-                    <t t-if="currentOrder.get_partner()" t-esc="currentOrder.get_partner().name"/>
-                    <t t-else="">Customer</t>
-                </span> 
-            </button>
-            <button class="button js_invoice btn btn-light py-3 text-start rounded-0 border-bottom" t-att-class="{ 'highlight text-bg-primary': currentOrder.is_to_invoice() }" 
-                t-on-click="toggleIsToInvoice">
-                <i class="fa fa-file-text-o me-2" />Invoice 
-            </button>
-            <button t-if="pos.config.iface_tipproduct and pos.config.tip_product_id" class="button js_tip btn btn-light py-3 text-start rounded-0 border-bottom" t-att-class="{ 'highlight text-bg-primary': currentOrder.get_tip() }" 
-                t-on-click="addTip">
-                <i class="fa fa-heart me-2" />Tip 
-                <t t-if="currentOrder.get_tip() != 0">
-                    (<t t-esc="env.utils.formatCurrency(currentOrder.get_tip())" />)
-                </t>
-            </button>
-            <button t-if="pos.config.iface_cashdrawer" class="button js_cashdrawer btn btn-light py-3 text-start rounded-0 border-bottom" 
-                t-on-click="openCashbox">
-                <i class="fa fa-archive me-2" />Open Cashbox 
-            </button>
-            <button t-if="pos.config.ship_later" class="button btn btn-light py-3 text-start rounded-0 border-bottom" t-att-class="{ 'highlight text-bg-primary': currentOrder.getShippingDate() }" 
-                t-on-click="toggleShippingDatePicker">
-                <i class="fa fa-clock-o me-2" />Ship Later 
-                <span t-if="currentOrder.getShippingDate()">
-                    (<t t-esc="currentOrder.getShippingDate()" />)
-                </span>
+    <t t-name="point_of_sale.PaymentScreenBack">
+        <t t-if="ui.isSmall">
+            <button class="btn-switchpane validation-button btn btn-secondary flex-fill py-4 rounded-0 fw-bolder fs-1 w-50"
+                t-on-click="() => pos.showScreen('ProductScreen')">
+                <span>Back</span>
             </button>
+        </t>
+        <t t-else="">
+            <div class="button back validation btn btn-light btn-lg py-5 rounded-0 d-flex flex-column align-items-center justify-content-center fw-bolder flex-grow-1 w-50"
+                t-on-click="() => pos.showScreen('ProductScreen')">
+                <div class="pay-circle d-flex align-items-center justify-content-center mb-2">
+                    <i class="fa fa-angle-double-left me-2" />
+                </div>
+                <span class="back_text">Back</span>
+            </div>
+        </t>
+    </t>
+
+    <t t-name="point_of_sale.PaymentScreenBottom">
+        <div t-attf-class="d-flex {{ui.isSmall ? 'switchpane' : 'flex-row'}}">
+            <t t-call="point_of_sale.PaymentScreenValidate" />
+            <t t-call="point_of_sale.PaymentScreenBack" />
         </div>
     </t>
```

Comment:
if this is only used once there's no need to have a separate template for it. You can inline it